### DISCUSSION TITLE
404 code in Wordpress Plugin wrong

### DISCUSSION THREAD

**Discussion by rossdav70** [plausible/analytics#4542] [Anything Else]
Working in Worpress I noticed that with 404 switched on in the plugin I was seeing my path properties all set to "documentation.location.pathname"

Looking at the front end there is a clear typo in the outputted JS

```html
<script type="text/javascript" id="plausible-analytics-js-after">
/* <![CDATA[ */
window.plausible = window.plausible || function() { (window.plausible.q = window.plausible.q || []).push(arguments) }
document.addEventListener('DOMContentLoaded', function () { plausible( '404', {"props":{"path":"documentation.location.pathname"}} ); });
/* ]]> */
</script>
```

To get it working I have disabled 404 in the plugin and instead manually added the correct script to my 404 template.

```html
<script type="text/javascript" id="plausible-analytics-js-after">
/* <![CDATA[ */
window.plausible = window.plausible || function() { (window.plausible.q = window.plausible.q || []).push(arguments) }
document.addEventListener('DOMContentLoaded', function () { plausible('404', {"props":{"path": document.location.pathname}}); });
/* ]]> */
</script>
```

**Comment by ruslandoga** [comment]
@rossdav70 👋 

Thank you for the report!

Seems like it happens here: https://github.com/plausible/wordpress/blob/8e6c39d7d65552ece47283504ccdd4308e3e1313/src/Actions.php#L74

---

@Dan0sz 👋 

I wonder if `documentation.location.pathname` should be "unquoted"?

**Reply by Dan0sz** [reply]
Yeah, that should definitely be the case.
I'll create an issue for the next release.

Thank for reporting! @rossdav70
